contributing.rst

```diff
@@ -1426,12 +1426,48 @@ Use the ``figure`` to markup an image with a caption.
 
     The rest of the indented content will be the caption. This can be a short sentence or several paragraphs. Captions can contain any other rst markup.
 
-.. _inline-images:
+.. _substitutions:
 
-Inline images
+Substitutions
 """""""""""""""
 
-You can add inline images in the document using `substitutions <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#substitution-definitions>`_. Substitution definitions are indicated by an explicit markup start (".. ") followed by a vertical bar, the substitution text (which gets substituted), another vertical bar, whitespace, and the definition block. A substitution definition block may contain inline-compatible directives such as :ref:`image <images>` or `replace <http://docutils.sourceforge.net/docs/ref/rst/directives.html#replace>`_. The following block of code substitutes arrow in the text with the image specified.  
+Substitutions are a useful way to define a value which is needed in many places. Substitution definitions are indicated by an explicit markup start (".. ") followed by a vertical bar, the substitution text (which gets substituted), another vertical bar, whitespace, and the definition block. A substitution definition block may contain inline-compatible directives such as :ref:`image <images>` or `replace <http://docutils.sourceforge.net/docs/ref/rst/directives.html#replace>`_. For more information, refer this `guide <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#substitution-definitions>`_.
+
+You can define the value once like this:
+
+.. code-block:: rst 
+
+  .. |RST| replace:: reStructuredText
+  
+and then reuse it like this:
+
+.. code-block:: rst 
+
+  We use |RST| to write documentation source files.
+  
+Here, ``|RST|`` will be replaced by reStructuredText
+
+You can also create a reference with styled text:
+
+.. code-block:: rst
+  
+  .. |slack| replace:: **ODK Slack**
+  .. slack: https://opendatakit.slack.com
+
+You can use the hyperlink reference by appending a "_" at the end of the vertical bars, for example:
+
+.. code-block:: rst
+
+  You can ask about your problem in |slack|_.
+
+.. |slack| replace:: **ODK Slack**
+.. _slack: https://opendatakit.slack.com  
+
+You can ask about your problem in |slack|_.
+
+The ``rst_epilog`` in :file:`conf.py` contains a list of global substitutions that can be used in any file. 
```

Comment:
Can you provide examples (in rst code-blocks and then, repeated, in regular text) of each of the defined substitutions?

Reply:
Sure!